lib/plugins/aws/deploy/compile/events/apiGateway/tests/methods.js

```diff
@@ -162,21 +162,25 @@ describe('#compileMethods()', () => {
   });
 
   it('should set the correct lambdaUri', () => {
-    const lambdaUri = `arn:aws:apigateway:${
-      awsCompileApigEvents.options.region}:lambda:path/2015-03-31/functions/arn:aws:lambda:${
-      awsCompileApigEvents.options.region}:12345678:function:${
-      awsCompileApigEvents.serverless.service.service}-${
-      awsCompileApigEvents.options.stage}-first/invocations`;
+    const lambdaUriObject = {
+      'Fn::Join': ['', [
+        'arn:aws:apigateway:',
+        { Ref: 'AWS::Region' },
+        ':lambda:path/2015-03-31/functions/',
+        { 'Fn::GetAtt': ['first', 'Arn'] },
+        '/invocations'  // eslint-disable-line comma-dangle
+      ]]                // eslint-disable-line comma-dangle
```

Comment:
You could try to write it in string representation (like we've done here: https://github.com/serverless/serverless/blob/v1.0/lib/plugins/aws/deploy/compile/events/schedule/index.js#L59-L71) and then parse it (https://github.com/serverless/serverless/blob/v1.0/lib/plugins/aws/deploy/compile/events/schedule/index.js#L86).

This way you could get rid of the linting errors without disabling the linter for those specific lines.